makeflow/src/makeflow.c

```diff
@@ -1370,7 +1370,7 @@ int main(int argc, char *argv[])
 				}
 				break;
 			case LONG_OPT_DISABLE_BATCH_CACHE:
-				cache_mode = 0;
+				cache_mode = 1;
```

Comment:
cache_mode = 1  should be with cache on, otherwise it gets confusing with the double negatives.